rvic/core/read_forcing.py

```diff
@@ -298,27 +298,22 @@ def read(self, timestamp):
         # Get the liquid fluxes
         log.info('getting fluxes for %s (%s)', timestamp, self.current_tind)
         forcing = {}
-        for i, fld in enumerate(self.liq_flds):
-            self.current_fhdl.variables[fld].set_auto_maskandscale(False)
-            if i == 0:
-                forcing['LIQ'] = \
-                    (self.fld_mult[fld] *
-                     self.current_fhdl.variables[fld][self.current_tind, :, :])
-            else:
-                forcing['LIQ'] += \
-                    (self.fld_mult[fld] *
-                     self.current_fhdl.variables[fld][self.current_tind, :, :])
+        for flux, fields in (('LIQ', self.liq_flds), ('ICE', self.ice_flds)):
```

Comment:
This is a CESM hold over, since the CESM CPL supports two routing tracers (liquid and ice).  It would not be too hard to rework the configuration file to support runtime assignment of those fields but right now, I'm not sure its worth it. I'm not aware of anyone routing tracers other than liquid using RVIC.